recipes/bw2speedups/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "bw2speedups" %}
+{% set version = "2.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "04cf55d9fdb74876fb690978966ca5901f2908a8d7607feb99b9e5fe361b8f72" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - numpy
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - bw2speedups
+
+about:
+  home: The package home page
+  license: BSD License
+  license_family: BSD
+  license_file: ''
+  summary: Cython functions to speed up Brightway2 LCA calculations
+  description: ''
+  doc_url: https://bitbucket.org/cmutel/brightway2-speedups
+  dev_url: https://bitbucket.org/cmutel/brightway2-speedups
+
+extra:
+  recipe-maintainers:
+    - cardosan
+    - cmutel
```

Comment:
@cmutel -- are you ok being added as a co-maintainer of the bw2speedups recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

Also, any thoughts on providing the `license_file` for this package? It could be included here via:
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

if you don't want to push a new release.

Reply:
@cardosan I released 2.2.1 with the license file included (filename LICENSE). 

@synapticarbors Thanks a lot for your help! Yes, I am OK being added as a co-maintainer.